docs/current/guides/snippets/build-local-images/export/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"dagger.io/dagger"
+)
+
+func main() {
+	// initialize Dagger client
+	ctx := context.Background()
+	daggerClient, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stdout))
```

Comment:
Nit: just `client` instead of `daggerClient` is simpler.

Reply:
Done